RELEASES.md

```diff
@@ -1,8 +1,22 @@
-# Release Process
-This document explains the process to release new version and to address patch release.
+# Releases
+This document explains the process to release new versions.
+
+## Versioning
+Versions are expressed as `vX.Y.Z` as well as Semantic Versioning;
+
+- `X` is the major version
+- `Y` is the minor version
+- `Z` is the patch version
```

Comment:
```suggestion
- `X` is the major version
- `Y` is the minor version <- current major release
- `Z` is the patch version <- current minor / patch release
```

Reply:
IMHO, we need to use the term "minor versions" for the current major versions (`Y`), so that everyone cannot be confused. Also, we need to use the term "patch versions" for the current minor / patch versions (`Z`). WDYT?